tests/patient.test.js

```diff
@@ -88,4 +93,45 @@ describe('Patient', function () {
       })
     })
   })
+
+  describe.only('Patient Route', () => {
+    
+    describe('Patients', () => {
+
+      it('GET all patients', function () {
+        return agent
```

Comment:
This promised needed to be returned, `done` was not being called that's why it was timing out